tensor/examples/datafs-sim/sim.go

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"math/rand/v2"
+	"reflect"
+	"strconv"
+
+	"cogentcore.org/core/base/errors"
+	"cogentcore.org/core/core"
+	"cogentcore.org/core/tensor"
+	"cogentcore.org/core/tensor/databrowser"
+	"cogentcore.org/core/tensor/datafs"
+	"cogentcore.org/core/tensor/stats/stats"
+)
+
+type Sim struct {
+	Root   *datafs.Data
+	Config *datafs.Data
+	Stats  *datafs.Data
+	Logs   *datafs.Data
+}
+
+// ConfigAll configures the sim
+func (ss *Sim) ConfigAll() {
+	ss.Root = errors.Log1(datafs.NewDir("Root"))
+	ss.Config = errors.Log1(ss.Root.Mkdir("Config"))
+	errors.Log1(datafs.New[int](ss.Config, "NRun", "NEpoch", "NTrial"))
+	ss.Config.Item("NRun").SetInt(5)
+	ss.Config.Item("NEpoch").SetInt(20)
+	ss.Config.Item("NTrial").SetInt(25)
```

Comment:
So are you moving things that used to be nice struct fields and making them arbitrary `datafs` entries? How does this relate to `econfig`/`cli`?

Reply:
in an actual sim, these will still be a struct, but for this simple demo, this was easier..